### ISSUE TITLE
Is there a way to use the plugin as a dimmer, not just a switch?

### ISSUE THREAD

**Issue by lz1kka** [AMoo-Miki/homebridge-tuya-lan#56]
Hello,

I bought wall dimmer that works with TuyaSmart app, but with this plugin I am not able to use it as a dimmer, but instead only as a standard switch (on/off). I see another plugin "homebridge-tuya" which has such an option, but it is listed as "depreciated". So could that be done with this one? I don't need any ColorTemperature or RGB features, just ON/OFF+Dimming

Thanks in advance!

**Comment by BR4DY** [comment]
Have you tried changing the type to:
                    "type": "RGBTWLight",

**Comment by Rantaholic** [comment]
Try this solution from a previously closed issue for the same question and possibly #37 at the bottom of that thread.

https://github.com/AMoo-Miki/homebridge-tuya-lan/issues/19#issuecomment-472334773

**Comment by lz1kka** [comment]
> Try this solution from a previously closed issue for the same question and possibly #37 at the bottom of that thread.
> 
> [#19 (comment)](https://github.com/AMoo-Miki/homebridge-tuya-lan/issues/19#issuecomment-472334773)

I have added "dp": "3" as attribute and everything seems to be fine! Thanks a lot for your help!

**Comment by AMoo-Miki** [comment]
I am glad you were able to use the, then unreleased, SimpleDimmer.

I have made a release of all the devices that were tested in the past couple of months. The documentation for [Supported Devices](https://github.com/AMoo-Miki/homebridge-tuya-lan/wiki/Supported-Devices#simple-heaters) and [Setup Instructions](https://github.com/AMoo-Miki/homebridge-tuya-lan/wiki/Setup-Instructions) have been updated.

The latest `rc` release appears to be stable enough to use; give it a shot by doing `npm i -g homebridge-tuya-lan@rc`. It includes logic to understand this newer API.
